Apps/WebAPI/sdLitica.WebAPI/Controllers/v1/BaseApiController.cs

```diff
@@ -11,6 +13,6 @@ public class BaseApiController: Controller
         /// current user id
         /// </summary>
         //public string UserId => HttpContext.User.FindFirstValue(ClaimTypes.NameIdentifier);
-        public string UserId => User.Identity.Name;        
+        public string UserId => ((Claim)((ClaimsIdentity)User.Identity).Claims.First()).Value.ToString();        
```

Comment:
cast to Claim is unnecessary

Reply:
Is there any possibilities for User.Identity returns null in this context? Or maybe Claims will be empty? 
For example, a future code mistake in calling UserId from non-authenticated request. So, since this code is on Base Class, and many usage will be available in future, a suggestion is to create a simple method and test these situations.

Reply:
Both are addressed. Made property more managable and removed surplus cast